components/metrics/BUILD.gn

```diff
@@ -181,143 +197,150 @@ if (!is_ios) {
   }
 }
 
-static_library("net") {
-  sources = [
-    "net/cellular_logic_helper.cc",
-    "net/cellular_logic_helper.h",
-    "net/net_metrics_log_uploader.cc",
-    "net/net_metrics_log_uploader.h",
-    "net/network_metrics_provider.cc",
-    "net/network_metrics_provider.h",
-    "net/wifi_access_point_info_provider.cc",
-    "net/wifi_access_point_info_provider.h",
-  ]
-
-  public_deps = [
-    ":metrics",
-  ]
-  allow_circular_includes_from = [ ":metrics" ]
+if (!use_cobalt_customizations) {
+  static_library("net") {
```

Comment:
@andrewsavage1 do you have a good hint or previously used trick to prevent if blocks that disable Cobalt build code like here to not cause large indentation diffs ? E.g. is there some attribute or something that could be added to the target to mark it as disabled or non-functional ?

Reply:
nope—my trick is to use an IDE with code folding unfortunately 

that being said, it looks like some of these conditionals disables can be grouped together as they're already adjacent

Reply:
I've grouped adjacent blocks under one if